src/helpers.rs

```diff
@@ -33,7 +34,6 @@ pub fn random_string() -> String {
 }
 
 pub fn hash(data: &[u8]) -> String {
-    println!("{}", dotenv::var("KEY").expect("Missing env `KEY`"));
     argon2::hash_encoded(
         data,
         random_string().as_bytes(),

```

Comment:
Same here, you should fetch the env var at the beginning of the execution and then pass them as parameters